WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -749,29 +734,32 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
     return [WPBlogTableViewCell cellHeight];
 }
 
-# pragma mark - UISearchController delegate methods
+# pragma mark - UISearchBar delegate methods
 
-- (void)willPresentSearchController:(UISearchController *)searchController
+- (void)searchBar:(UISearchBar *)searchBar textDidChange:(NSString *)searchText
 {
     self.dataSource.searching = YES;
```

Comment:
I think we need to move this line into `searchBarTextDidBeginEditing:`. With the old implementation, as soon as the search bar was focussed the dataSource went into 'search mode', so that all hidden sites become visible. With this PR currently, the sites list only updates when you start typing.